puppet-userdata/outputs.tf

```diff
@@ -1,3 +1,4 @@
 output "user_datas" {
-  value = ["${data.template_file.metadata_puppet.*.rendered}"]
+  value = [data.template_file.metadata_puppet.*.rendered]
```

Comment:
Are the explicit list brackets needed here?

Reply:
afaik no